scripts/generate_gff3_db.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
Most GFF3 source will most likely require it's own conversion
script.  There is a lot of variation in sources.  This will be
particularly important when we start implementing more complex
name queries.

We need to work towards having a library that can be programmed
against to make developing these programs  straight forward.

Jerome Kelleher <notifications@github.com> writes:
> In scripts/generate_gff3_db.py:
> 
> > @@ -0,0 +1,127 @@
> > +"""
> 
> Presumably this script will need to be rolled into the repo manager, right? We
> want to say, add-feature-set and the repo manager parses the GFF file and
> creates the SQLlite DB. The repository administrator shouldn't need to know
> anything about our internal DB format for storing features.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>